CHANGELOG.md

```diff
@@ -1,5 +1,41 @@
 # Changelog
 
+## Unreleased release/0.11.0 <Badge text="beta" type="success"/>
+
+These changes are available in the [release/0.11.0 branch](https://github.com/PrefectHQ/prefect).
```

Comment:
Either now or on master these need to be moved down to the [master branch] changelog section